manual/core/README.md

```diff
@@ -231,34 +231,35 @@ See [AccessibleByName] for an explanation of the conversion rules.
+| CQL3 data type     | Getter name    | Java type            | See also                          |
+|--------------------|----------------|----------------------|-----------------------------------|
+| ascii              | getString      | java.lang.String     |                                   |
+| bigint             | getLong        | long                 |                                   |
+| blob               | getByteBuffer  | java.nio.ByteBuffer  |                                   |
+| boolean            | getBoolean     | boolean              |                                   |
+| counter            | getLong        | long                 |                                   |
+| date               | getLocalDate   | java.time.LocalDate  | [Temporal types](temporal_types/) |
+| decimal            | getBigDecimal  | java.math.BigDecimal |                                   |
+| double             | getDouble      | double               |                                   |
+| duration           | getCqlDuration | [CqlDuration]        | [Temporal types](temporal_types/) |
+| float              | getFloat       | float                |                                   |
+| inet               | getInetAddress | java.net.InetAddress |                                   |
+| int                | getInt         | int                  |                                   |
+| list               | getList        | java.util.List<T>    |                                   |
+| map                | getMap         | java.util.Map<K, V>  |                                   |
+| set                | getSet         | java.util.Set<T>     |                                   |
+| smallint           | getShort       | short                |                                   |
+| text               | getString      | java.lang.String     |                                   |
+| time               | getLocalTime   | java.time.LocalTime  | [Temporal types](temporal_types/) |
+| timestamp          | getInstant     | java.time.Instant    | [Temporal types](temporal_types/) |
+| timeuuid           | getUuid        | java.util.UUID       |                                   |
+| tinyint            | getByte        | byte                 |                                   |
+| tuple              | getTupleValue  | [TupleValue]         | [Tuples](tuples/)                 |
+| user-defined types | getUDTValue    | [UDTValue]           | [User-defined types](udts/)       |
+| uuid               | getUuid        | java.util.UUID       |                                   |
+| varchar            | getString      | java.lang.String     |                                   |
+| varint             | getBigInteger  | java.math.BigInteger |                                   |
+| vector             | getVector      | [CqlVector]          | [Custom Codecs](custom_codecs/)   |
```

Comment:
The "custom_codecs" link refers to another manual page; all the links at the bottom point directly at Javadoc.  I thought about adding a local ref to the URL but note that we don't do that with any of the other value types here (including CqlDuration, which serves as something of our model).

As for updating the links to point at the current release: that's something we do as part of the release process.  We haven't done it the last few times around because the doc updates were delayed for various reasons (mostly build-related) that we believe are now resolved.

This conversation certainly does highlight that CqlVector needs to have a link at the bottom; I'll add that for sure.